src/data_generation.jl

```diff
@@ -1,3 +1,5 @@
+using Random
```

Comment:
let's keep all imports in `src/ChainRulesTestUtils`

Reply:
Also, `Random` needs to be moved to the `[deps]` in Project.toml (and removed from `[extras]` and `[targets]`)